scripts/scil_assign_custom_color_to_tractogram.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+The script use scalar in data_per_point and data_per_streamline
+(e.g commit_weights) to visualise on the streamlines.
+Saves the RGB values in the data_per_point (color_x, color_y, color_z).
+
+If called with .tck, the output will always be .trk, because data_per_point has
+no equivalent in tck file.
+
+The usage of --use_dps, --use_dpp and --from_anatomy is more complex. It maps
+the raw values from these sources to RGB using a colormap.
+    --use_dps: nbr. streamlines = len(streamlines)
+    --use_dpp: nbr. points = len(streamlines._data)
```

Comment:
I don't understand this. First, please avoid to use the '.' 
You mean number of streamlines = len(streamlines) ?
number of points ?
streamlines._data -> what does that mean?